src/lib/components/comparisonTable/index.stories.mdx

```diff
@@ -77,21 +77,34 @@ export const ComparisonTableWithData = () => {
           id: 4,
           key: 'imdb',
           label: 'IMDB rating',
+          render: (value) => (
+            <TableButton onClick={() => {}}>
+              {value}
+            </TableButton>
+          ),
         },
         {
           id: 3,
           key: 'rating',
-          label: <TableRowHeader label="Our Rating" onClickInfo={console.log} />,
+          label: <TableRowHeader label="Our rating of movies found online on IMDB lists" onClickInfo={() => {}} />,
```

Comment:
If you change the label here, please also update it in the "pseudo code" that's shown for consistency.

Reply:
<img width="670" alt="image" src="https://user-images.githubusercontent.com/13664983/218419746-15c51322-ea1c-49b0-ad81-8ad09a909ea8.png">